lib/data/recruitment_banners.yml

```diff
@@ -0,0 +1,80 @@
+# Example usage of adding a banner to the banners list:
+
+  # - name: Banner 1
+  #   suggestion_text: "Help improve GOV.UK"
+  #   suggestion_link_text: "Sign up to take part in user research (opens in a new tab)"
+  #   survey_url: https://google.com
+  #   page_paths: 
+  #     - /
+  #     - /foreign-travel-advice
+
+banners:
+  - name: check-employment-status-for-tax Banner
```

Comment:
I'm wondering about the`name` key. It's not being used for anything right? So I suppose it's to help map the banner to the banner request we receive. So in this case they would all be HMRC team banner as per the [Trello description](https://trello.com/c/eV6vhoUa/2300-govuk-user-research-banner-requests-hmrc-team-set-up-m) card.